README.md

```diff
@@ -1534,6 +1534,7 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 - [iodine](https://code.kryo.se/iodine/) - IPv4 over DNS tunnel solution, enabling you to start up a socks5 proxy listener. ([Source Code](https://github.com/yarrick/iodine)) `ISC` `C`
 - [microproxy](https://github.com/thekvs/microproxy) - lightweight non-caching HTTP/HTTPS proxy server. `MIT` `Go`
 - [Nginx Proxy Manager](https://nginxproxymanager.com/) - Nginx Proxy Manager is an easy way to accomplish reverse proxying hosts with SSL termination. ([Source Code](https://github.com/jc21/nginx-proxy-manager)) `MIT` `Nodejs/Docker`
+- [Outline Server](https://getoutline.org/) - A proxy server that runs a Shadowsocks instance for each access key and a REST API to manage the access keys. ([Source Code](https://github.com/Jigsaw-Code/outline-server)) `Apache-2.0` `Docker/Nodejs`
```

Comment:
The `VPN` section would be a better fit, but we delegated it to https://github.com/awesome-foss/awesome-sysadmin#vpn which is currently unmaintained. I'm ok with adding it to `Proxy` until we figure out what should be done with the awesome-sysadmin list https://github.com/awesome-selfhosted/awesome-selfhosted/issues/2482